plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -283,7 +283,7 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
             _production_enabled = true;
 
 
-         if( fc::time_point::now() - block->timestamp < fc::seconds(5) || (block->block_num() % 1000 == 0) ) {
+         if( fc::time_point::now() - block->timestamp < fc::minutes(5) || (block->block_num() % 1000 == 0) ) {
```

Comment:
This doesn't really belong in this PR but if we are going to modify it we should probably make it configurable so that the _user_ can decide what is appropriate for logging.

Reply:
It belongs to this PR in a way that if you receive only irreversible blocks your head block time is always delayed for more then 5 seconds (in case of 21 block producers). So, it's a cheat suggested by Dan.